public/app/plugins/panel/gettingstarted/components/TutorialCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC, MouseEvent } from 'react';
+import { GrafanaTheme } from '@grafana/data';
+import { Icon, stylesFactory, useTheme } from '@grafana/ui';
+import { css } from 'emotion';
+import store from 'app/core/store';
+import { cardContent, cardStyle, iconStyle } from './sharedStyles';
+import { Card } from '../types';
+
+interface Props {
+  card: Card;
+}
+
+export const TutorialCard: FC<Props> = ({ card }) => {
+  const theme = useTheme();
+  const styles = getStyles(theme, card.done);
+
+  return (
+    <a
+      className={styles.card}
+      target="_blank"
+      onClick={(event: MouseEvent<HTMLAnchorElement>) => handleTutorialClick(event, card)}
+    >
+      <div className={cardContent}>
+        <div className={styles.type}>{card.type}</div>
+        <div className={styles.heading}>{card.done ? 'complete' : card.heading}</div>
+        <h4>{card.title}</h4>
+        <div className={styles.info}>{card.info}</div>
+        <Icon className={iconStyle(theme, card.done)} name={card.icon} size="xxl" />
+      </div>
+    </a>
+  );
+};
+
+const handleTutorialClick = (event: MouseEvent<HTMLAnchorElement>, card: Card) => {
+  event.preventDefault();
+  const isSet = store.get(card.key);
+  if (!isSet) {
+    store.set(card.key, true);
+  }
+  window.open(card.href, '_blank');
```

Comment:
Aren't ad blockers going to block this?

Reply:
I don't know, are they?

Reply:
if you have a onClick & href & target="_blank" does the onClick trigger?

Reply:
It seems to work at least. But can remove the target prop from a element